Kwc/Basic/LinkTag/Extern/Component.php

```diff
@@ -17,4 +17,24 @@ public static function getSettings($param = null)
         return $ret;
     }
 
+    public function getTemplateVars(Kwf_Component_Renderer_Abstract $renderer)
+    {
+        $ret = parent::getTemplateVars($renderer);
+
+        $row = $this->_getRow();
+        $rel = $this->getData()->rel ? $this->getData()->rel : array();
+        if ($row->rel_nofollow) {
+            $rel[] = 'nofollow';
+        }
+        if ($row->rel_noopener) {
+            $rel[] = 'noopener';
+        }
+        if ($row->rel_noreferrer) {
+            $rel[] = 'noreferrer';
+        }
+        $ret['rel'] = implode(' ', array_unique($rel));
```

Comment:
is this rel used? in abstract component tpl there is only $this->data->rel used not $this->rel?